srtcore/queue.h

```diff
@@ -135,8 +135,8 @@ friend class CRcvBuffer;
 
 struct CSNode
 {
-   CUDT* m_pUDT;		// Pointer to the instance of CUDT socket
-   uint64_t m_llTimeStamp_tk;      // Time Stamp
+   CUDT* m_pUDT;               // Pointer to the instance of CUDT socket
+   uint64_t m_llTimeStamp_tk;  // Next send event Time Stamp
```

Comment:
Ok, I clarified it a bit. No, this change of the field name in CRNode was intentional because it has also changed its meaning and logical interpretation. Previously this was holding the time that was read at the moment of moving this node at the end of queue, and the "time for event" was determined by adding 100ms to it. Now adding 100ms happens at the moment when moving this node, so it doesn't hold the time of the moving ("Time Stamp"), but the time when it should be picked up.